File: library/src/main/java/com/lazygeniouz/house/ads/listener/AdListener.java
/*
 * Created by Darshan Pandya.
 * @itznotabug
 * Copyright (c) 2018.
 */

package com.lazygeniouz.house.ads.listener;

public interface AdListener {

    void onAdLoadFailed(Exception e);
    void onAdLoaded();
    void onAdClosed();
    void onAdShown();
    void onApplicationLeft();
}

File: library/src/main/java/com/lazygeniouz/house/ads/listener/AdListener.kt
/*
 * Created by Darshan Pandya. (@itznotabug)
 * Copyright (c) 2018-2020.
 */

package com.lazygeniouz.house.ads.listener

interface AdListener {
    fun onAdLoaded()
    fun onAdClosed()
    fun onAdShown()
    fun onApplicationLeft()
    fun onAdFailedToLoad(exception: Exception)
}
